src/Illuminate/Console/Scheduling/ScheduleRunCommand.php

```diff
@@ -128,7 +139,19 @@ public function handle(Schedule $schedule, Dispatcher $dispatcher, Cache $cache,
                 $this->newLine();
             }
 
-            if ($event->onOneServer) {
+            if ($this->dryRun) {
+                $summary = $event->getSummaryForDisplay();
```

Comment:
Since all the code in here is duplicated below, would extracting to a protected method be more maintainable?